frontend/src/app/components/project-list/project-list.component.ts

```diff
@@ -29,13 +29,17 @@ export class ProjectListComponent implements OnInit, OnDestroy {
   public itemsShow = this.projects;
   public unsubscribe$ = new Subject<void>();
 
-  constructor(public projectService: ProjectService,
+  constructor(
+    public projectService: ProjectService,
     private createProjectService: CreateProjectService, 
     private currentOrganization: GetCurrentOrganizationService,
     private currentUserService: GetCurrentUserService) {
   }
 
   ngOnInit(): void {
+    this.subscribeToCurrentUser();
+    this.subscribeToCurrentOrganization();
+
     this.projectService.getAllProjectsOfThisOrganization(this.currentOrganizationId)
```

Comment:
You should put it in the c#`this.subscribeToCurrentOrganization();` to be sure that it is loaded correctly

Reply:
Added these edits